src/test/java/com/openshift/client/utils/QuickstartTestUtils.java

```diff
@@ -27,6 +27,8 @@
  * @author André Dietisheim
  */
 public class QuickstartTestUtils {
+    
+    private QuickstartTestUtils() {}
```

Comment:
+1 but the constructor should not precceed the constants and identation is wrong.